src/main/resources/createTable

```diff
@@ -0,0 +1,16 @@
+--creating table
+CREATE TABLE REGISTRATION(id INTEGER not NULL,first VARCHAR(255),last VARCHAR(255),age INTEGER,PRIMARY KEY (id));
+--populating table
+INSERT INTO Registration VALUES (100, 'Zara', 'Ali', 18);
+INSERT INTO Registration VALUES (101, 'Mahnaz', 'Fatma', 25);
+INSERT INTO Registration VALUES (102, 'Zaid', 'Khan', 30);
+INSERT INTO Registration VALUES (103, 'Sumit', 'Mittal', 28);
+
+
+--creating table
+CREATE TABLE EMPLOYEES(id INTEGER not NULL,first VARCHAR(255),last VARCHAR(255),age INTEGER,PRIMARY KEY (id));
```

Comment:
Strange titles. What does it mean EMPLOYEE? REGISTRATION ?

Reply:
This is a sample data, while we do not have actual models. Could be changed to actual data like(USER, FLIGHT etc.) right now or after we add models to the project.

Reply:
I can review DB scheme and then you add correct entities, ok ?

Reply:
Will change it to correct entities right after DB scheme will be accepted, should I add all entities from DB scheme?

Reply:
Moved Connection and Statement  open\close  from constructor to method. Removed printAllDb method. As EugeneGrim requested.

Reply:
I can propose to create new task for creation correct data. If @EugeneGrim agree with this, he can resolve this discussion and merge this branch.